ironic-deployment/ironic/ironic.yaml

```diff
@@ -4,6 +4,10 @@ metadata:
   name: capm3-ironic
 spec:
   replicas: 1
+  strategy:
+    # Ironic cannot run with more than one replica at a time, so prevent this
```

Comment:
Let's please reword this, ironic can run with a lot of replicates, *we* cannot run it this way.

Reply:
Right, will do!

Reply:
Better?